SimPEG/Maps.py

```diff
@@ -373,6 +383,254 @@ def nP(self):
         return self._nP
 
 
+class SelfConsistentEffectiveMedium(IdentityMap, properties.HasProperties):
```

Comment:
I am wondering if we want to start a `SimPEG.Emperical` for these sorts of things (and create a mapping based on that)